src/components/Composer/index.android.tsx

```diff
@@ -1,100 +1,94 @@
-import PropTypes from 'prop-types';
-import React, {useCallback, useEffect, useMemo, useRef} from 'react';
-import {StyleSheet} from 'react-native';
-import _ from 'underscore';
+import React, {ForwardedRef, useCallback, useEffect, useMemo, useRef} from 'react';
+import {StyleProp, StyleSheet, TextInput, TextStyle} from 'react-native';
 import RNTextInput from '@components/RNTextInput';
 import * as ComposerUtils from '@libs/ComposerUtils';
 import themeColors from '@styles/themes/default';
 
-const propTypes = {
+type ComposerProps = {
     /** Maximum number of lines in the text input */
-    maxLines: PropTypes.number,
+    maxLines: number;
 
     /** If the input should clear, it actually gets intercepted instead of .clear() */
-    shouldClear: PropTypes.bool,
-
-    /** A ref to forward to the text input */
-    forwardedRef: PropTypes.func,
+    shouldClear: boolean;
 
     /** When the input has cleared whoever owns this input should know about it */
-    onClear: PropTypes.func,
+    onClear: () => void;
 
     /** Set focus to this component the first time it renders.
      * Override this in case you need to set focus on one field out of many, or when you want to disable autoFocus */
-    autoFocus: PropTypes.bool,
+    autoFocus: boolean;
 
     /** Prevent edits and interactions like focus for this input. */
-    isDisabled: PropTypes.bool,
+    isDisabled: boolean;
 
     /** Selection Object */
-    selection: PropTypes.shape({
-        start: PropTypes.number,
-        end: PropTypes.number,
-    }),
+    selection: {
+        start: number;
+        end?: number;
```

Comment:
Based on previous `propTypes`, both values are not required.

Reply:
This is the react native type from `TextInput.d.ts`:

`selection?: {start: number; end?: number | undefined} | undefined;`

Do you think we should change it?

Reply:
Okay then. Let's keep it as it is.